src/pages/UsersManagement/UsersManagement.tsx

```diff
@@ -1,32 +1,48 @@
 import Button, { AppearanceTypes } from 'components/molecules/Button/Button'
-import { useFetchUsers } from 'hooks/requests/useUsers'
+import { useDownloadUsers, useFetchUsers } from 'hooks/requests/useUsers'
 import { useTranslation } from 'react-i18next'
 import { FC } from 'react'
 import AddedUsersTable from 'components/organisms/tables/AddedUsersTable/AddedUsersTable'
 import classes from './classes.module.scss'
-import { isEmpty } from 'lodash'
+import { isEmpty, includes } from 'lodash'
 import classNames from 'classnames'
 import { Root } from '@radix-ui/react-form'
 import Tooltip from 'components/organisms/Tooltip/Tooltip'
 import UserManagementCheatSheet from 'components/molecules/cheatSheets/UserManagementCheatSheet'
+import useAuth from 'hooks/useAuth'
 
 const UsersManagement: FC = () => {
-  const { users, handelFilterChange, handelSortingChange } = useFetchUsers()
+  const {
+    users,
+    paginationData,
+    handelFilterChange,
+    handelSortingChange,
+    handlePaginationChange,
+  } = useFetchUsers()
   const { t } = useTranslation()
+  const { userPrivileges } = useAuth()
+  const { downloadCSV, isLoading } = useDownloadUsers()
+
+  const handelDownloadFile = () => {
+    downloadCSV()
+  }
 
   return (
     <>
       <div className={classes.userManagementHeader}>
         <h1>{t('users.user_management')}</h1>
-        {/*  {// TODO: add toolTip here  */}
         <Tooltip
           title={t('cheat_sheet.user_management.title')}
           modalContent={<UserManagementCheatSheet />}
         />
         <Button
-          // href="/settings/users/add"
           appearance={AppearanceTypes.Secondary}
-          className={classNames({ [classes.invisible]: isEmpty(users) })}
+          className={classNames({
+            [classes.invisible]: !includes(userPrivileges, 'EXPORT_USER'),
```

Comment:
Let's use hidden prop for hiding the button.

Reply:
yes, hidden prop is usually the best option, but currently I wanted to make it invisible not remove it.